atoMEC/models.py

```diff
@@ -22,7 +22,7 @@
 from . import convergence
 from . import writeoutput
 from . import xc
-from . import unitconv
+from postprocess import pressure
```

Comment:
```suggestion
from atoMEC.postprocess import pressure
```
This fixes the docs build problem
```
WARNING: autodoc: failed to import module 'models' from module 'atoMEC'; the following exception was raised:
No module named 'postprocess'
```
Sphinx does not know about the `postprocess` module, it tries to import the module from the Python Standard lib and fails. One can fix this either via absolute imports (my suggestion) or relative imports, see https://realpython.com/absolute-vs-relative-python-imports/ for an overview.

Reply:
I follow your suggestion ;)